tests/test_colors.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+test_colors
+Test for colors 
+@author: iancharest
+"""
+
+import unittest
+from pyrsa.vis.colors import color_scale
+from pyrsa.vis.colors import rdm_colormap
+
+class ColorTests(unittest.TestCase):
+
+    def test_color_scale(self):
+        n_cols = 10
+        cols = color_scale(n_cols)
+        n_cols_returned, n_rgb = cols.shape
+        self.assertEqual(n_cols_returned, n_cols)
+        self.assertEqual(n_rgb, 3)
+
+    def test_rdm_colormap(self):
+        n_cols = 10
+        cols = rdm_colormap(n_cols)
+        n_cols_returned, n_rgb = cols.colors.shape
+        self.assertEqual(n_cols_returned, n_cols)
+        self.assertEqual(n_rgb, 3)
+
+if __name__ == '__main__':
+    unittest.main()     
```

Comment:
this part is not necessary I think, if you want to run one test case you can call:

 `python setup.py test -s tests.test_colors.ColorTests`

Reply:
i've added a test case. the last colour returned from rdm_colormap should by design be yellow.